pkg/ttnpb/end_device.go

```diff
@@ -1553,8 +1553,7 @@ func (m *EndDevice) ExtractRequestFields(dst map[string]interface{}) {
 	m.GetIds().ExtractRequestFields(dst)
 }
 
-// Utility
-
+// UpdateTimestamps sets correct CreatedAt and UpdatedAt timestamps for EndDevice based on src device.
```

Comment:
Specifically correct here means that the `CreateAt` is the earliest timestamp, while the `UpdatedAt` is the latest timestamp.